docs/METRICS.md

```diff
@@ -1,47 +1,22 @@
+
 ## Firefox Screenshots Metrics
+*Last Update: 2017-03-21*
 
-A summary of the metrics Firefox Screenshots will record, and what we're looking for in those metrics.
+This document is a summary of the metrics Firefox Screenshots will record, how we're recording them, and what we're looking for in those metrics.  There are two main areas we'll look at:
 
-### Opt-out
+0. Metrics to tell us if Firefox Screenshots is affecting Firefox.  For instance, do Firefox Screenshots users engage longer with Firefox?
```

Comment:
We did have a talk with Brendan Colloran about this, and he was pretty confident that we couldn't find any meaningful way to understand this, except through controlled experiment.  That is, Firefox Screenshots users are going to use Firefox more, because people who use any feature are more active than the general population.

The one option – which I'd like us to consider at some point – is to actually do a controlled experiment by leaving some set of users with Screenshots disabled.  And maybe we could see some differences for the brief period when we are rolling out Screenshots (but probably not).